src/dapp/libexec/dapp/dapp

```diff
@@ -6,6 +6,47 @@
 ### Build, test, deploy and transact with Ethereum contracts from the comfort of your command line.
 ###
 
+OPTS="dapp [<options>] <command> [<args>]
+dapp <command> --help
+--
+  Compilation options:
+optimize             activate solidity optimizer
+legacy               compile using the '--combined-json' format
+extract              after building, write the .abi, .bin and .bin-runtime. Implies '--legacy'
+
+  Testing options:
+verbosity=<number>   verbosity of 'dapp test' output (0-3)
+v,verbose            sets verbosity to 1
```

Comment:
shouldn't we have `-vv` and `-vvv` here?

Reply:
maybe worth a note in the changelog

Reply:
I ended up putting them back in anyway for now